src/modules/integrator-guide/pages/compose-configuration/page-blocks.adoc

```diff
@@ -743,3 +743,263 @@ Defines the report the page bock will use when showing data.
 Defines the display element from the selected report the page block will use when showing data.
 
 |===
+
+[#page-block-navigation]
+== Navigation
+
+The *navigation page block* allows you to define custom navigation inside your {PRODUCT_NAME} {APP_NAME_COMPOSE} applications applications. The navigation can be an addition to the pre-defined one or a complete replacement.
+
+[TIP]
+====
+You can define multiple navigation items for the navigation page blocks, which allows you to customize and display all supported navigation item type.
+====
+
+image:compose-configuration/page-block-reference/config-navigation.png[...,49%,role="preview data-zoomable"]
+image:compose-configuration/page-block-reference/added-navigation.png[...,49%,role="preview data-zoomable"]
+
+.Additional configurations:
+[cols="2s,5a"]
+|===
+| [#page-block-navigation-appearance]#<<page-block-navigation-appearance,Appearance>>#
+|
+Defines the style of the navigation
+
+.Supported appearances:
+* Tabs +
+
+[annotation,role="data-zoomable"]
+----
+{
+  "image": {
+    "rel": "compose-configuration/page-block-reference/added-navigation-tabs.png",
+    "alias": "compose-configuration-page-block-reference-added-navigation-tabs",
+    "w": 1920,
+    "h": 1080
+  },
+  "view": {
+    "x": 321,
+    "y": 0,
+    "h": 400,
+    "w": 1599
+  },
+  "annotations": [{
+    "kind": "box-note",
+    "x": 333,
+    "y": 68,
+    "h": 53,
+    "w": 400
```

Comment:
Your alignment is off; yours looks like this

![image](https://github.com/cortezaproject/corteza-docs/assets/12545711/024ea07f-5ef6-403e-80b2-db64b58af70c)

but should be like this

![image](https://github.com/cortezaproject/corteza-docs/assets/12545711/d25a03c3-2e5e-4edc-9885-cfeb444b85d6)

Use the red box to wrap (tightly) around the bit you're trying to annotate.
The built-in style guide then takes care of the rest.


**!!** you don't need to be pixel perfect; close enough will do